samples/ingestion/ingestion-client/Connector/Serializable/TextAnalytics/TextAnalyticsRequestsChunk.cs

```diff
@@ -1,19 +0,0 @@
-// <copyright file="TextAnalyticsRequestsChunk.cs" company="Microsoft Corporation">
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
-// </copyright>
-
-namespace Connector.Serializable
-{
-    using System.Collections.Generic;
-
-    public class TextAnalyticsRequestsChunk
```

Comment:
Not sure why this is being removed.

Reply:
seems to be a duplication, classes are not referenced anymore